frontend/templates/product/sections/ProductSection/ProductOptions.tsx

```diff
@@ -210,14 +210,7 @@ function ProductOptionImage({ image }: ProductOptionImageProps) {
          : null;
 
    if (ratio == null) {
-      return (
-         <Img
-            h="16"
-            src={image.url}
-            alt={image.altText ?? ''}
-            objectFit="contain"
-         />
-      );
+      return <Img h="16" src={image.url} alt="" objectFit="contain" />;
```

Comment:
I'm curious why the `alt` is always an empty string now?

Reply:
This only applies to product options, and I think that it's better for the screen reader to focus on the label (e.g. "Part Only"), rather than on the image.. Setting `alt` to `""` is akin to say that the image is only for decorative purposes

<img width="437" alt="Screenshot 2022-10-18 at 16 45 43" src="https://user-images.githubusercontent.com/4640135/196463559-3d8e1939-2b80-444f-a65a-16a575ff3df0.png">